README.md

```diff
@@ -1,5 +1,19 @@
 # Learning TypeScript Projects
 
+<!-- Dev Badge Section -->
+<p align="center">
+  <a href="https://github.com/LearningTypeScript/projects/actions/workflows/prettier.yml">
+    <img alt="Code Style: Prettier" src="https://img.shields.io/badge/code_style-prettier-14cc21.svg">
+  </a>
+  <a href="https://github.com/LearningTypeScript/projects/actions/workflows/markdown.yml">
+    <img alt="Markdown Style: Markdownlint" src="https://img.shields.io/badge/markdown_style-markdownlint-14cc21.svg">
+  </a>
+  <a href="https://github.com/LearningTypeScript/projects/actions">
+    <img alt="Github Actions: Build Passing" src="https://img.shields.io/badge/github_actions-build_passing-14cc21.svg">
+  </a>
+  <img alt="TypeScript: Strict" src="https://img.shields.io/badge/typescript-strict-14cc21.svg">
+</p>
```

Comment:
Hmm, these badges look great alone, but it's a little odd seeing them center-aligned when both the heading and existing all-contributors badge are left-aligned:

![Screenshot of these new center-aligned badges between the left-aligned header and left-aligned existing all-contributors badge](https://user-images.githubusercontent.com/3335181/170990117-8839db09-ace6-4435-9bdc-e2334b357b86.png)

I don't really have a preference for centered vs. left-aligned. So as long as everything goes with the same alignment, whichever way you like more / is less of a pain would work.

Reply:
I didn't want to touch the contributors badge.. since it stated not to modify 😬

Reply:
Yeah those comments are pretty aggressive haha. I'm betting you can wrap _around_ them in some way... or failing that, `badgeTemplate` from https://allcontributors.org/docs/en/bot/configuration.

Reply:
https://github.com/all-contributors/all-contributors/issues/361

Reply:
oh looks like I have to edit the `badgeTemplate`. I think this repo did something cool, maybe it's variables in markdown or so.. https://github.com/squirrellyjs/squirrelly/blob/master/README.md